helix-view/src/view.rs

```diff
@@ -154,6 +187,11 @@ impl View {
             object_selections: Vec::new(),
             gutters,
             doc_revisions: HashMap::new(),
+            search_matches: SearchMatches {
+                matches: Vec::new(),
+                new: Regex::new("").unwrap(),
+                old: Regex::new("").unwrap(),
```

Comment:
i assume `unwrap` is safe to use in a case like this? but it is a big assumption...